bindings/rust/bench/certs/generate_certs.sh

```diff
@@ -62,13 +63,13 @@ cert-gen () {
 
 if [[ $1 != "clean" ]] 
 then
-    cert-gen ec 384
-    cert-gen rsa 2048
-    cert-gen rsa 3072
-    cert-gen rsa 4096
+    cert-gen ec 384 ecdsa384
```

Comment:
What if the script took ecdsa284 as an input and parsed the input into the strings that it needed :)

Reply:
I thought about parsing or just doing a simple `if` statement, but manually specifying the directory name is the simplest to understand and most extensible for say if another signature algorithm was added.